lib/Yars/Routes.pm

```diff
@@ -82,6 +82,13 @@ sub _get {
           || _redirect_to_remote_stash( $c, $filename, $md5 )
           || $c->reply->not_found;
     };
+
+    if ($c->req->headers->header('X-Use-X-Accel'))
+    {
+        $c->res->headers->add('X-Accel-Redirect', "$dir/$filename");
```

Comment:
I don't mind that, but I don't see how it can conflict.  Right now, it is considered a full, absolute path to the file on the physical file system.  If you mounted your Yars disk at "/status" or "/version", it would still work fine.  (Not that I would recommend that..)